.github/workflows/ci-dgraph-load-tests.yml

```diff
@@ -16,13 +16,18 @@ on:
 jobs:
   dgraph-load-tests:
     if: github.event.pull_request.draft == false
-    runs-on: self-hosted
+    runs-on: [self-hosted, x64]
     steps:
       - uses: actions/checkout@v3
+      - name: Get Go Version
+        run: |
+          #!/bin/bash
+          GOVERSION=$({ [ -f .go-version ] && cat .go-version; })
+          echo "GOVERSION=$GOVERSION" >> $GITHUB_ENV
```

Comment:
+1 ^ thanks for adding this, could you also do this for `go-lint`?

Reply:
@skrdgraph Sure, previously the linter yaml was skipping the Go installation step entirely with `skip-go-installation: true` .  I just pushed changes that model the linter after the [Badger linter setup](https://github.com/dgraph-io/badger/blob/main/.github/workflows/ci-golang-lint.yml) (hence the version bumps).  Let me know if that looks okay.